utils.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Obv utility to run the `cb` once, as soon as the condition given by
+ * `filter` is true.
+ */
+function onceWhen(obv, filter, cb) {
```

Comment:
There's another discussion to be had one day, but it's not important for this PR, which is: should we keep using Obv? I find it rather weak in API compared to other observable libraries out there, and there are certainly good ones that have decent API, are very minimal in bundle size, and performant. But for now, let's just keep using Obv.

Reply:
I wouldn't mind changing to something else than Obz. Fixed a pretty bad bug in it in the beginning of db2 that has been there for ages. I think it's one of those, build by dominic and thus used a lot in ssbc, but not much elsewhere.

Reply:
Since my other response was eaten by /dev/null. Yes